.travis.yml

```diff
@@ -13,10 +13,10 @@ install:
 
 script:
   - "./.travis/tests.sh"
+  - "./.travis/build.sh"
```

Comment:
you could run this build.sh script in an `after_success:` block, just after `script:` (only run if the latter is successful)